src/sentry/incidents/logic.py

```diff
@@ -163,7 +164,7 @@ def update_incident_status(
         incident.update(**kwargs)
 
         if status == IncidentStatus.CLOSED:
-            create_incident_snapshot(incident, windowed_stats=True)
+            create_pending_incident_snapshot(incident)
```

Comment:
I can't comment on the code I want since it's hidden, but a couple lines up we delete the `IncidentSnapshot` if the incident re-opens. Should probably delete the pending snapshot too?